bin/wp-now

```diff
@@ -13,7 +13,5 @@ if [[ "$@" != *"--path="* ]]; then
 fi
 
 cd $SCRIPT_DIR/..
-export NVM_DIR=$HOME/.nvm;
-source $NVM_DIR/nvm.sh;
-nvm use
+
```

Comment:
Do we need to have a node version check? What happens if your default node version is insufficient?

Reply:
> Do we need to have a node version check?

I consider this bin script as a wrapper, a helper that allows running commands similarly to when installed via npm. I wouldn't add any validation logic here, then. 

> What happens if your default node version is insufficient?

In such a case, I would expect behavior to be the same as for a user who runs `npx nx preview wp-now` without a wrapper.

Reply:
+1 for removing `nvm` dependency – it prevents people don't use `nvm` from running this script even if they still have the required Node.js version. @luisherranz expects that to be the case at WCEU workshop